## Introduction ##
The availability of a large number of sequenced bacterial genomes allows researchers not
only to derive functional and regulation information about specific organisms but also to study the
fundamental properties of the organization of a genome. Here we address an important and challenging
question regarding the global arrangement of operons in a bacterial genome: why operons
in a bacterial genome are arranged in the way they are. We have previously studied this question
and found that operons of more frequently activated pathways tend to be more clustered together
in a genome. Specifically, we have developed a simple sequential distance-based pseudo energy function
and found that the arrangement of operons in a bacterial genome tend to minimize the clusteredness
function (C value) in comparison with artificially-generated alternatives, for a variety
of bacterial genomes. Here we extend our previous work, and report a number of new observations:
(a) operons of the same pathways tend to group into a few clusters rather than one; and (b) the
global arrangement of these operon clusters tend to minimize a new ‘‘energy’’ function (C+ value)
that reflects the efficiency of the transcriptional activation of the encoded pathways. These obser-
11 vations provide insights into further study of the genomic organization of genes in bacteria.

## Citing us ##
Qin Ma, Ying Xu, Global genomic arrangement of bacterial genes is closely tied with the total transcriptional efficiency. _Genomics, Proteomics & Bioinformatics_, 2013, 10.1016/j.gpb.2013.01.004

## Contact ##
Any questions, problems, bugs are welcome and should be dumped to Qin Ma <**_qin.ma@sdstate.edu_**>
